### PR TITLE
Fixed Wish triggering Disguise

### LINKED ISSUE #5856: Disguise ability triggered from getting hit by Wish (=non damaging)
### Description

My Mimikyu took damage from my Jirachi's wish, it triggered the disguise ability.

https://github.com/user-attachments/assets/849b514d-73fb-49ea-a2e7-0cf5656adb1b

### Version

1.9.4

### Upcoming/master Version

_No response_

### Discord contact info

wiz1989

### PR DESCRIPTION
**Pull request by AsparagusEduardo**
## Issue(s) that this PR fixes
Fixes #5856 

## **Discord contact info**
AsparagusEduardo

### CODE REVIEW

File: src/battle_script_commands.c [hunk 1]
```diff
@@ -14465,6 +14465,7 @@ static void Cmd_trywish(void)
             gBattleMoveDamage = max(1, GetNonDynamaxMaxHP(gBattlerAttacker) / 2);
         }
 
+        gHitMarker |= HITMARKER_IGNORE_DISGUISE;
```
Comment (AlexOn1ine) [inline src/battle_script_commands.c:14468]: shouldn't you clear that somewhere as well?

Comment (AsparagusEduardo) [reply src/battle_script_commands.c:14468]: It's cleared during `HandleAction_ActionFinished`

**Review by AlexOn1ine — DISMISSED** [review]

**Comment by AlexOn1ine** [comment]
sorry, could you add the hitmaker in the script instead together with the sub hitmaker?

**Comment by AsparagusEduardo** [comment]
> sorry, could you add the hitmaker in the script instead together with the sub hitmaker? Eh, it doesn't matter much. Just wanted have them to be in one place. Up to you

Done 👍

**Review by AlexOn1ine — APPROVED** [review]

**Pull request merged**